doc/conf.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+#
+# ZConfig documentation build configuration file, created by
+# sphinx-quickstart on Thu Jan 12 14:36:41 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx']
```

Comment:
I hate to nitpick [*], but

```python
extensions = [
    'sphinx.ext.autodoc',
    'sphinx.ext.intersphinx',
]
```

[*] who am I kidding, I love to nitpick, especially about unimportant style questions